co.headsetapp.headset.yml

```diff
@@ -0,0 +1,45 @@
+id: co.headsetapp.headset
+base: org.electronjs.Electron2.BaseApp
+base-version: '19.08'
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: headset
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --env=TMPDIR=/var/tmp
+  - --talk-name=org.freedesktop.Notifications
+  - --own-name=org.mpris.MediaPlayer2.headset
```

Comment:
After trying the app, it doesn't seems to use mpris at all or this is supposed to work?

Reply:
It seems to work fine with`org.mpris.MediaPlayer2.chromium.instance3` you might need to change that somewhere at the source code of your app. By default your app owns the name of `org.mpris.MediaPlayer2.$app-id`

Reply:
I've tried installing the latest test build on Ubuntu 18.04 and 20.04 and everything works without any problem there, including mpris.
When using D-Feet, I can only see `org.mprsi.MediaPlayer2.headset`. The app has explicit code to expose the name I've set in the manifest, and also explicitly blocks any attempt by electron to create additional session busses.
A reliable tool to control mpris interfaces (besides using dbus command line) is [playerctl](https://github.com/altdesktop/playerctl). I've found that some OSs intercept keyboard shortcuts, and mess up mpris. Also using [Mpris Indicator Button](https://extensions.gnome.org/extension/1379/mpris-indicator-button/) is another great way to control mpris players. In fact, that gnome extension has specific code with exceptions to make shuffle and repeat work out of the box.
What distro are you using?

Reply:
@bilelmoussaoui actually, now that you mention it, there was a problem that has already been fixed upstream with mpris. As I mentioned, we now have code to block electron from automatically creating its own mpris session, which the current version in flatpak doesn't have. It's the same with the icons, upstream uses scalable icons, but the version in flatpak doesn't, hence why I added them explicitly.

Reply:
Sure, it would be nice to use the current(the one set by electron) mpris name as well and remove it once you have published a new release later, lgtm otherwise

Reply:
I can't add the electron mpris name to the manifest because electron uses a random number `org.mpris.MediaPlayer2.chromium.instance${random number}` and even if I add it with an asterisk (which probably is not supported) might cause trouble with other electron apps that haven't deal carefully with mpris since I'm basically calling any mpris electron mpris name mine.

Reply:
Makes sense. Thanks for the details